enterprise/coderd/coderd.go

```diff
@@ -275,6 +275,19 @@ func New(ctx context.Context, options *Options) (_ *API, err error) {
 			r.Delete("/organizations/{organization}/members/roles/{roleName}", api.deleteOrgRole)
 		})
 
+		r.Group(func(r chi.Router) {
+			r.Use(
+				apiKeyMiddleware,
+				httpmw.ExtractOrganizationParam(api.Database),
+				// Intentionally using ExtractUser instead of ExtractMember.
+				// It is possible for a member to be removed from an org, in which
```

Comment:
It's desirable because the alternative is to forcibly terminate workspaces when a user is removed. A workspace should remain in case:
1. Removing the user was an accident
2. There is data in the workspace that is valuable before termination

I think we do have some dormant workspace behavior today to eventually delete this. Ideally there is some page of "orphaned" workspaces an admin could go through and clean these up.

But this PR does not address any of that. It just makes sure the `quota` api call still works in this case. Otherwise the `/{workspace}` page would fail to load because of this failed api call.